packages/task/src/browser/task-service.ts

```diff
@@ -994,20 +994,19 @@ export class TaskService implements TaskConfigurationClient {
         }
 
         // Create / find a terminal widget to display an execution output of a task that was launched as a command inside a shell.
-        const widget = await this.taskTerminalWidgetManager.open(
-            <TaskTerminalWidgetOpenerOptions>{
-                created: new Date().toString(),
-                taskId,
-                id: this.getTerminalWidgetId(processId),
-                title: taskInfo
-                    ? `Task: ${taskInfo.config.label}`
-                    : `Task: #${taskId}`,
-                destroyTermOnClose: true,
-                widgetOptions: { area: 'bottom' },
-                mode: widgetOpenMode,
-                taskConfig: taskInfo ? taskInfo.config : undefined
-            }
-        );
+        const widget = await this.taskTerminalWidgetManager.open({
+            created: new Date().toString(),
+            id: this.getTerminalWidgetId(processId),
+            title: taskInfo
+                ? `Task: ${taskInfo.config.label}`
+                : `Task: #${taskId}`,
+            destroyTermOnClose: true
+        }, {
+            taskId,
+            widgetOptions: { area: 'bottom' },
+            mode: widgetOpenMode,
+            taskConfig: taskInfo ? taskInfo.config : undefined
+        });
```

Comment:
@akosyakov I am breaking one object into two, and only pass the `TerminalWidgetFactoryOptions` into the terminal widget. Is this the right way to do it? Thank you !

Reply:
yes, it looks good.

Reply:
yes, it looks good.